packages/dataviews/src/components/dataform/index.tsx

```diff
@@ -11,7 +11,27 @@ import { DataFormProvider } from '../dataform-context';
 import { normalizeFields } from '../../normalize-fields';
 import { DataFormLayout } from '../../dataforms-layouts/data-form-layout';
 
-export default function DataForm< Item >( {
+/**
+ * Loops through the list of data items and returns an object with the intersecting ( same ) key and values.
+ *
+ * @param data list of items.
+ */
+function getIntersectingValues< Item extends object >( data: Item[] ): Item {
+	const intersectingValues = {} as Item;
+	const keys = Object.keys( data[ 0 ] ) as Array< keyof Item >;
+	for ( const key of keys ) {
+		const [ firstRecord, ...remainingRecords ] = data;
+		const intersects = remainingRecords.every( ( item ) => {
+			return item[ key ] === firstRecord[ key ];
+		} );
+		if ( intersects ) {
+			intersectingValues[ key ] = firstRecord[ key ];
+		}
```

Comment:
Should we add `else intersectingValues[ key ] = Symbol( 'mixed' )` or something to allow controls to handle "mixed" values properly.